### PR TITLE
Updated README aurorawatch: Alex

### LINKED ISSUE #122: Update aurorawatch pipeline README
As a developer, I want clear documentation that outlines the installation process, the structure of the pipeline, and how to run the app, so new contributors can easily get started and understand the functionality of the app.

- Edit README so that it is up to date with aurorawatch pipeline progress

### PR DESCRIPTION
**Pull request by AlexOBarnes**
This closes #122

I have updated the readme to include the following information on:
- Setup of pipeline
- Usage of pipeline
- How each component works

### CODE REVIEW

File: aurorawatch-pipeline/README.md [hunk 1]
```diff
@@ -2,7 +2,84 @@
...
+source venv/bin/activate
+pip install -r requirements.txt
+```
+3. Create a `.env` file with the following:
+    - DB_HOST - to access your RDS instance
+    - DB_USER - username for accessing RDS
+    - DB_PASSWORD - password for accessing RDS
+    - DB_NAME - the name of your database
+    - DB_PORT - port to access RDS (typically 5432 for postgres)
+
```
Comment (olliejth) [inline aurorawatch-pipeline/README.md:21]: Very useful to have this in

**Review by CalebMathers — APPROVED** [review]

File: aurorawatch-pipeline/README.md [hunk 2]
```diff
@@ -2,7 +2,84 @@
...
+4. Verify your credentials:
+```bash
+aws configure
+```
+This will require sensitive information to be entered through the command line.
+
+5. Add to your `.env` file the following:
+    - REGION - region of your ECR repository
+    - ECR_URI - endpoint of the ECR repository
+    - IMAGE_NAME - a name given to your docker image
```
Comment (olliejth) [inline aurorawatch-pipeline/README.md:41]: REGION should be "AWS_REGION"

File: aurorawatch-pipeline/README.md [hunk 3]
```diff
@@ -2,7 +2,84 @@
...
+
+6. Run the following command to automatically dockerise and push to ECR:
+```bash
+bash ../dashboard/deploy.sh
+```
+7. Now follow the terraform setup steps in the `../terraform` folder to provision a lambda based off this docker image.
 
 ## Usage
+- To use locally one can use the following command:
```
Comment (olliejth) [inline aurorawatch-pipeline/README.md:50]: Indeed

File: aurorawatch-pipeline/README.md [hunk 4]
```diff
@@ -2,7 +2,84 @@
...
+python aurora_pipeline.py
+```
+It is a good idea to ensure that the pipeline works with your credentials prior to provisioning the AWS architecture.
+
+- To run tests run the following command:
+```bash
+pytest
+```
+Alternatively to run specific tests or test files:
+```bash
```
Comment (olliejth) [inline aurorawatch-pipeline/README.md:61]: Nice that you've given the user options.

File: aurorawatch-pipeline/README.md [hunk 5]
```diff
@@ -2,7 +2,84 @@
...
+#### `aurora_load.py`
+- Utilises private environment variables using `os` and `python-dotenv` libraries.
+- Loads the processed data using `psycopg2`.
+#### `aurora_pipeline.py`
+- Orchestrates the ETL pipeline and configures logging.
+#### `conftest.py`
+- Contains `pytest.fixture` functions for tests in this folder.
+#### `test_aurora_[filename]`
+- Contains the tests for each module in this folder.
+- Uses `pytest` and `unittest.mock`.
```
Comment (olliejth) [inline aurorawatch-pipeline/README.md:85]: Good level of detail for this readme.

**Pull request merged**